lib/tagify_string/core_ext.rb

```diff
@@ -26,11 +26,22 @@ def tagify(options = {})
     end
 
     # The Sonia Clause.  Check for all garbage characters.  Return an empty string in this case.
-    output = self.gsub("_", " ").gsub("'", "").parameterize(opts[:sep])
+    output = ""
+
+    if Rails::VERSION::MAJOR >= 5
+      output = self.gsub("_", " ").gsub("'", "").parameterize(separator: opts[:sep])
+    else
+      output = self.gsub("_", " ").gsub("'", "").parameterize(opts[:sep])
+    end
+    
     return "" if output.length == 0
 
     # Otherwise, attach prefix, process for case and send back.
-    output = [prefix, output].join(" ").parameterize(opts[:sep]) if prefix.present?
+    if Rails::VERSION::MAJOR >= 5
+      output = [prefix, output].join(" ").parameterize(separator: opts[:sep]) if prefix.present?  
```

Comment:
talvez fique melhor ter a validação do 'prefix.present?' and de vermos a validação do Rails::VERSION::MAJOR, mas por mim passa